### PR TITLE
Mount NFS without restarting VM (fixes Issue#3)

### PR DESCRIPTION
**Pull request by aambekar-box**

### CODE REVIEW

File: minikube-nfs.sh [hunk 1]
```diff
@@ -309,9 +309,6 @@ sudo umount -i /Users'
 sudo mkdir -p \""$shared_folder"\""
   done
 
-  bootlocalsh="${bootlocalsh}
-sudo /usr/local/etc/init.d/nfs-client start"
```
Comment (kunalparmar) [inline minikube-nfs.sh:313]: This is required for the new nfs settings to take effect in case the VM is rebooted

**Review by kunalparmar — CHANGES_REQUESTED** [review]

File: minikube-nfs.sh (hunk 1 above)
Comment (kunalparmar) [reply minikube-nfs.sh:313]: Never mind, this file does not exist in a minikube VM
```bash
$ ls /usr/local/etc/init.d/nfs-client
ls: cannot access '/usr/local/etc/init.d/nfs-client': No such file or directory
```

File: minikube-nfs.sh [hunk 2]
```diff
@@ -329,18 +326,15 @@ sudo mount -t nfs -o "$prop_mount_options" "$prop_nfshost_ip":\""$shared_folder"
...
 
-  echoInfo "Stopping machine ...\n"
-  minikube stop -p $prop_profile_name
-  echoSuccess "\t\t\t\t\t\t\tOK"
-
-  echoInfo "Starting machine ...\n"
-  minikube start -p $prop_profile_name
+  echoInfo "Mounting NFS ...\n"
+  minikube ssh -p $prop_profile_name \
+  "sudo /var/lib/boot2docker/bootlocal.sh"
```
Comment (kunalparmar) [inline minikube-nfs.sh]: Let's move this one line to `configureBoot2Docker` and remove `restartMachine` altogether

**Review by kunalparmar — CHANGES_REQUESTED** [review]
Can you:
* make the suggested change
* reference issue #3 in your commit
* run the repro steps from issue #3 and report as a comment on this pull request

File: minikube-nfs.sh (hunk 2 above)
Comment (aambekar-box) [reply minikube-nfs.sh]: Updated.

**Comment by aambekar-box** [comment]
$ minikube start -p upgrade-issue --kubernetes-version=1.14.0
    😄  minikube v1.2.0 on darwin (amd64)
    💿  Downloading Minikube ISO ...
     129.33 MB / 129.33 MB [============================================] 100.00% 0s
    🔥  Creating virtualbox VM (CPUs=2, Memory=2048MB, Disk=20000MB) ...
    🐳  Configuring environment for Kubernetes v1.14.0 on Docker 18.09.6
    💾  Downloading kubeadm v1.14.0
    💾  Downloading kubelet v1.14.0
    🚜  Pulling images ...
    🚀  Launching Kubernetes ... 
    ⌛  Verifying: apiserver proxy etcd scheduler controller dns
    🏄  Done! kubectl is now configured to use "upgrade-issue"
    $ ./minikube-nfs.sh -p=upgrade-issue
    [INFO] Configuration:
    
            - Profile Name: upgrade-issue 
            - Shared Folder: /Users 
            - Mount Options: noacl,async,nfsvers=3 
            - Force: false 
    
    [INFO] Checking if machine is running ...               OK 
    [INFO] Lookup mandatory properties ...                  OK 
    
            - Machine IP: 192.168.99.252 
            - Network ID: vboxnet0 
            - NFSHost IP: 192.168.99.1 
    
    [INFO] Configure NFS ... 
    
     !!! Sudo will be necessary for editing /etc/exports !!! 
    Password:
                                                            OK 
    [INFO] Configuring machine...                           OK 
    [INFO] Mounting NFS ...
                                                            OK 
    [INFO] Verify NFS mount ...                             OK 
    
    --------------------------------------------
    
     The minikube profile 'upgrade-issue'
     is now mounted with NFS!
    
     ENJOY high speed mounts :D
    
    --------------------------------------------
    
    $ kubectl version --short | grep Server
    Server Version: v1.14.0

**Review by kunalparmar — APPROVED** [review]

**Pull request merged**